### PR TITLE
Adds arabic translation

### PR DESCRIPTION
**Pull request by owayss**
Adds Arabic translation.

### CODE REVIEW

**Comment by lorenzleutgeb** [comment]
Maybe cross-check with #46

**Comment by owayss** [comment]
@mrassili would you kindly CR this?

File: site/ar.njk [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+meta_title: نداء للحد من جائحة كوفيد-١٩
+meta_description: الحكومات لن تستطيع بمفردها حل المشكلة. على كل فرد منّا أن يقوم بواجبه. الله يخليك خليك بالبيت
```
Comment (mrassili) [inline site/ar.njk:4]: this is Egyptian dialect

File: site/ar.njk [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            <div class="flex justify-center pb-12">
+                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-size="large" data-text="I will do my part in the fight against #COVID19. I urge you to do the same!" data-url="https://staythefuckhome.com" data-hashtags="StayTheFuckHome" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+                <iframe class="ml-6" src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fstaythefuckhome.com&layout=button_count&size=large&appId=764658637392408&width=111&height=28" width="111" height="28" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media"></iframe>
+            </div>
+
+            <p class="text-sm text-gray-500 mb-10 mt-10">
+                إلى كل من يرغب في المساهمة في حراك وموقع StayTheFuckHome#، لقد قمنا بجعل الموقع مفتوح المصدر، فباستطاعة الجميع المساهمة.
+                كود الموقع موجود في العنوان التلي:  <a href="https://github.com/flore2003/staythefuckhome" target="_blank">https://github.com/flore2003/staythefuckhome</a>
```
Comment (mrassili) [inline site/ar.njk]: التالي
```suggestion
                كود الموقع موجود في العنوان التالي:  <a href="https://github.com/flore2003/staythefuckhome" target="_blank">https://github.com/flore2003/staythefuckhome</a>
```

File: site/ar.njk (hunk 1 above)
Comment (owayss) [reply site/ar.njk:4]: It is actually a reference to a popular folk song by the Lebanese singer Fairuz (https://en.wikipedia.org/wiki/Fairuz) that is now being used as a hashtag to encourage self-quarantine across the Arab world: https://twitter.com/hashtag/%D8%AE%D9%84%D9%8A%D9%83_%D8%A8%D8%A7%D9%84%D8%A8%D9%8A%D8%AA

I saw that other language translations maintained the original English hashtag #StayTheFuckHome where the hashtag was used, and figured it is best to keep using the site's original version for consistency.

Comment (mrassili) [reply site/ar.njk:4]: great! I like it.

**Review by flore2003 — APPROVED** [review]

**Comment by ahunts** [comment]
@owayss @mrassili @lorenzleutgeb Would you like to be on our attribution page when it is finished? If so, what name would you want us to use?

**Comment by lorenzleutgeb** [comment]
> @owayss @mrassili @lorenzleutgeb Would you like to be on our attribution page when it is finished? If so, what name would you want us to use?

Please don't mention me. Thanks.

**Comment by mrassili** [comment]
> @owayss @mrassili @lorenzleutgeb Would you like to be on our attribution page when it is finished? If so, what name would you want us to use?

Hey @ahunts Thanks a lot. Let's use `Marouane Rassili`

**Comment by owayss** [comment]
> @owayss @mrassili @lorenzleutgeb Would you like to be on our attribution page when it is finished? If so, what name would you want us to use?

Yes, you can use my name. It is "Owayss Kabtoul". Thanks:)

**Pull request merged**